rolecall/rolecall.py

```diff
@@ -101,73 +46,166 @@ class RoleCall:
     def __init__(self, bot):
         self.bot = bot
         self.settings = dataIO.load_json(SETTINGS_PATH)
+        self.reaction_queue = {}
+        self.reaction_user_queue = set()
+        self.queue_processor_task = bot.loop.create_task(self.queue_processor())
```

Comment:
to be nice and tidy, you might want to `__unload` this task as well

Reply:
implemented in commit 336c2f1